catroid/src/androidTest/java/org/catrobat/catroid/test/utils/TestUtils.java

```diff
@@ -92,4 +97,76 @@ public static Pixmap createRectanglePixmap(int width, int height, Color color) {
 		pixmap.fillRectangle(0, 0, width, height);
 		return pixmap;
 	}
+
+	public static Project createProjectWithGlobalValues(String name, String spriteName, String valueName, Context context) {
```

Comment:
move those methods in the test class.
TestUtils should be reserved for very general methods that are used in many test classes spanning multiple packages.
if you need this in multiple tests, its also better to create a new utils class in the test package where you need this and make every method package private.